frontend/common/src/fakeData/fakeExperienceSkills.ts

```diff
@@ -0,0 +1,24 @@
+import faker from "faker";
+import { Skill, ExperienceSkill, Experience } from "../api/generated";
+
+const generateExperienceSkill = (
+  skill: Skill,
+  experience: Experience,
+): ExperienceSkill => {
+  faker.setLocale("en");
+  return {
+    id: "blank",
```

Comment:
Not a big deal but you could fake this ID, too.

Reply:
> 

one of my refactoring broke it & roll backed it. It should be good now